### DISCUSSION TITLE
Datagrid panel not in list of Visualizations

### DISCUSSION THREAD

**Discussion by SunSparc** [grafana/grafana#70468] [Dashboards, Panels & Visualizations]
I just upgraded Grafana to version `10.0.0`. After reading about the new [Datagrid Visualization](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/datagrid/), I wanted to try it out. I created a new dashboard, then a new panel. I selected `--Grafana--` as the datasource. I opened the list of Visualizations and searched for "datagrid". The response is: "Could not find anything matching your query".

I search the Plugins page and see "Datagrid" by Grafana Labs is "`installed`".

How exactly does one begin to use the new "Datagrid" visualization?

**Comment by mbtx2** [comment]
This can be enabled through feature toggles.  https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/
Just add to the appropriate section in grafana.ini and restart.
